2-mandatory/3-bush-berries.js

```diff
@@ -22,7 +22,16 @@
 */
 
 function isBushSafe(berryArray) {
-  //Write your code here
+ 
+if (berryArray.every(x => x === "pink") === true){
```

Comment:
The comparison to true isn't needed as `berryArray.every(x => x === "pink")` returns true or false.